### ISSUE TITLE
Sophie no Atelier Fushigina Hon no Renkinjutsushi - CUSA02920

### ISSUE THREAD

**Issue by Lusiel0912** [red-prig/fpps4-game-compatibility#1320]
# Game name
Sophie no Atelier Fushigina Hon no Renkinjutsushi

## Code
CUSA02920

## Game version
1.00

## Used emulator's version
red-prig/fpPS4@224254a

## Current status
Nothing

## Error
```
[main:44596] nop nid:libSceGnmDriver:8ED92A5E900E63AC:sceGnmSetGsRingSizes
``` 

## Description
downloaded the libSceGnmDriver and put it into my game sce_moduel but not worked

## Screenshots
![image](https://user-images.githubusercontent.com/115441832/222918649-0691c2c9-aead-484e-a49a-8ebeb49180bd.png)

**Comment by ArtemVideoGames** [comment]
Don't use not whitelisted libs

**Comment by ArbestRi** [comment]
Blacklisted libs actually do stuff

**Comment by Kimieggie** [comment]
> Blacklisted libs actually do stuff

yes they do but reports with non whitelisted libs are deemed as invalid
